CodePush.ios.js

```diff
@@ -126,72 +126,106 @@ function checkForUpdate() {
  * releases, and displaying a standard confirmation UI to the end-user
  * when an update is available.
  */
-function sync(options = {}) {  
+function sync(options = {}, syncStatusChangeCallback, downloadProgressCallback) {  
   var syncOptions = {
-    descriptionPrefix: " Description: ",
-    appendReleaseDescription: false,
     
     ignoreFailedUpdates: true,
-    
-    mandatoryContinueButtonLabel: "Continue",
```

Comment:
Nevermind - I was under the impression that the `updateDialog` is a `boolean` flag, not the real dialog. In this case, the code you pointed me to earlier does just what I asked.